README.md

```diff
@@ -1 +1,16 @@
-### My_test
+# my_test
+
+A new Flutter project.
```

Comment:
Add a section to setup the local development environment.

Reply:
I updated the Readme file. Please Review this
![Readme](https://user-images.githubusercontent.com/69233556/143388218-0faa7a96-9e7e-4954-85c0-dcce32dac961.png)